pkg/config/config_test.go

```diff
@@ -5,7 +5,7 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
-	datadog "github.com/zorkian/go-datadog-api"
+	"github.com/zorkian/go-datadog-api"
```

Comment:
Someone in pod-dev has an IDE that actually _adds_ the package name here. Curious if there's a reason to remove it

Reply:
Yeah, I have gone back and forth on this. I get messages about unnecessary alias when it is present, but `gomock` doesn't work unless the alias is present (in my experience). Maybe we should go back to having `datadog` import alias present in all files where necessary?

Reply:
Yeah, I have gone back and forth on this. I get messages about unnecessary alias when it is present, but gomock doesn't work unless the alias is present (in my experience). Maybe we should go back to having datadog import alias present in all files where necessary?

Reply:
Made the alias `ddapi` everywhere instead of using the same name as the package (`datadog`) which fixes both issues I was having. The mockgen command should work without issue now for anyone without editing the files like I had to do before.